python/fledge/services/core/api/common.py

```diff
@@ -170,7 +170,7 @@ def do_shutdown(request):
         loop = request.loop
         asyncio.ensure_future(server.Server.shutdown(request), loop=loop)
     except RuntimeError as e:
-        _logger.error("Error while stopping Fledge server: {}".format(str(e)))
+        _logger.error(e, "Error while stopping Fledge server.")
```

Comment:
I think e is string here. (RuntimeError )

Reply:
Nope, To get multiline stacktrace we need Exception type.